src/VisualStudio/Core/Test.Next/Services/ServiceHubServicesTests.cs

```diff
@@ -1480,6 +1480,77 @@ internal async Task TestSourceGenerationExecution_DocumentChange_ButExternalUpda
         }
     }
 
+    [Theory, CombinatorialData]
+    internal async Task TestSourceGenerationExecution_RazorGeneratorAlwaysRuns_OtherGeneratorsRespectPreference(SourceGeneratorExecutionPreference executionPreference)
```

Comment:
Random aside: it's not clear to me why these tests are in ServiceHubServicesTests.cs. You're not doing what isn't already there, but I'm still confused.